README.markdown

```diff
@@ -40,10 +40,9 @@ Step by step install With Visual Studio 2017 for Windows 7/10 64 Bit
     Other two processes are similar to this one.
 * Install OpenGL with GLFW3.0 or later
   * Follow the tutorial in [this website](https://learnopengl.com/). Do not forget to install GLAD after installation of GLFW. It won't     cost you too much time.
-  * tips: You can download Windows pre-compiled binaries as an alternative. Click "64-bit Windows binaries" to download it. 
-* Install Balser Pylon 5
-  * Click [here](https://www.baslerweb.com/en/sales-support/downloads/software-downloads/) and download pylon Camera Software Suite.
-  * Check this [link](https://www.baslerweb.com/fp-1476182890/media/downloads/documents/application_notes/AW00136801000_Getting_Started_with_pylon5_and_OpenCV.pdf) to get started with         OpenCV and pylon.
+  * tips: the glad.c file is decided by your OpenGL version and something else, so the glad.c file which in the repository may not           match your case. Remember to replace it with your own file. It is important!! 
```

Comment:
I would like to rephrase the sentence as "Download the correct glad.c based on your OpenGL version and GPU and replace the original _glad.c_ file. (and provide a link.)" 

Is my memory correct? We have to make this clear. **something** is not a good word to use in the constructions.

Reply:
I must apologize for my imprecise statement. It is only based on OpenGL version. I will correct it immediately.